tooling/cli/index.bin.ts

```diff
@@ -111,9 +111,9 @@ OPTIONS:
 const version = () => console.log(pkgJson?.version);
```

Comment:
Please don't change unrelated files. I mean, this PRs for add new icons (Just do that) and don't changes anything.